Source/Streamstone.Tests/Scenarios/Tracking_entity_changes.cs

```diff
@@ -577,15 +586,21 @@ yield return
             }
         }
 
-        void InsertTestEntity(TestEntity entity)
+        void InsertTestEntity(ITableEntity entity)
```

Comment:
Ok, I documented why in the test [here](https://github.com/yevhen/Streamstone/blob/master/Source/Streamstone.Tests/Scenarios/Tracking_entity_changes.cs#L64)